app/app.go

```diff
@@ -241,6 +241,11 @@ func NewOsmosisApp(
 		app.BlockedAddrs(),
 	)
 
+	// TODO: There is a bug here, where we register the govRouter routes in InitNormalKeepers and then
+	// call setupHooks afterwards. Therefore, if a gov proposal needs to call a method and that method calls a
+	// hook, we will get a nil pointer dereference error due to the hooks in the keeper not being
+	// setup yet. I will refrain from creating an issue in the sdk for now until after we unfork to 0.47,
+	// because I believe the concept of Routes is going away.
 	app.SetupHooks()
```

Comment:
Would appreciate an issue linked (our side) prior to merge

Please document the decision to wait until the unfork

Reply:
https://github.com/osmosis-labs/osmosis/pull/6547/commits/ae09dbd75c60408c1dd169513ee850aa9cb65634